openmeter/subscription/repo/subscriptionrepo.go

```diff
@@ -29,6 +29,23 @@ func NewSubscriptionRepo(db *db.Client) *subscriptionRepo {
 	}
 }
 
+func (r *subscriptionRepo) MarkEdited(ctx context.Context, id models.NamespacedID, at time.Time) (subscription.Subscription, error) {
+	return entutils.TransactingRepo(ctx, r, func(ctx context.Context, repo *subscriptionRepo) (subscription.Subscription, error) {
+		ent, err := repo.db.Subscription.UpdateOneID(id.ID).SetLastEditedAt(at.UTC()).Where(dbsubscription.Namespace(id.Namespace)).Save(ctx)
```

Comment:
move where close to the beginning

```suggestion
		ent, err := repo.db.Subscription.UpdateOneID(id.ID).Where(dbsubscription.Namespace(id.Namespace)).SetLastEditedAt(at.UTC()).Save(ctx)
```

Reply:
the current order follow sql syntax order